### PR TITLE
add skyr-url/1.13.0 + fix static runtime

### PR DESCRIPTION
**Pull request by prince-chrismc**
Specify library name and version:  **lib/skyr-url**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/skyr-url/all/conanfile.py [hunk 1]
```diff
@@ -47,21 +47,25 @@ def config_options(self):
...
+            raise ConanInvalidConfiguration("apple-clang currently does not support with filesystem")
+
         if self.options.shared:
-            raise ConanInvalidConfiguration("shared is currently not supported")
+            # https://github.com/cpp-netlib/url/blob/dd345361ed86e4c1cabfe94743a8e769b346840c/src/CMakeLists.txt#L17
+            raise ConanInvalidConfiguration("shared is currently not supported by upstream")
+
+        if tools.Version(self.version) >= "1.13.0" and not self.settings.compiler == "Visual Studio":
+            # There's tedious compilation errors in C3i against ranges-v3
+            raise ConanInvalidConfiguration("{}/{} with {} is currently not supported".format(self.name, self.version, self.settings.compiler))
```
Comment (prince-chrismc) [inline recipes/skyr-url/all/conanfile.py:68]: If someone needs other platforms they will sadly have to figure it out themselves

**Review by SSE4 — APPROVED** [review]

**Review by theirix — APPROVED** [review]

**Review by madebr — APPROVED** [review]

File: recipes/skyr-url/all/conanfile.py (hunk 1 above)
Comment (madebr) [reply recipes/skyr-url/all/conanfile.py:68]: Does upstream know about the problem?

Comment (SpaceIm) [reply recipes/skyr-url/all/conanfile.py:68]: I don't know what was the issue, but it's perfectly fine with apple-clang. And you should disable `with_fs` by default for apple-clang.

Comment (prince-chrismc) [reply recipes/skyr-url/all/conanfile.py:68]: Given how much has changed in the last year I am not surprised.

**Pull request merged**